.github/workflows/tests.yaml

```diff
@@ -36,6 +36,10 @@ jobs:
             dependencies: "lowest"
           - php-version: "8.1"
             dependencies: "highest"
+          - php-version: "8.2"
+            dependencies: "lowest"
```

Comment:
Will this install Pimcore 10? 
If yes, I don't think we should add it, because Pimcore 10 doesn't support PHP 8.2. 
If no, we need to install the `pimcore/admin-ui-classic-bundle` (currently only for `highest`).

Reply:
Jep, you are right "Locking pimcore/pimcore (v10.5.1)"

Reply:
i remove that PHP 8.2 lowest part from the matrix